backend/corpora/lambdas/api/v1/curation/collections/common.py

```diff
@@ -187,9 +196,7 @@ class EntityColumns:
         "filename",
     ]
 
-    dataset_processing_status_cols = [
-        "processing_status",
-    ]
+    dataset_processing_status_cols = ["processing_status", "validation_message", "validation_status"]
```

Comment:
we should update the open API spec schema for datasets if validation_message and validation_status are going to be included in the response